spec.html

```diff
@@ -37172,6 +37172,7 @@
       <h1>agent-order</h1>
       <p>For a candidate execution _execution_, _execution_.[[AgentOrder]] is a Relation that satisfies the following conditions.</p>
       <emu-alg>
+        1. Let agent-order be _execution_.[[AgentOrder]].
```

Comment:
I don't like the addition either, but I wasn't sure what to change it to. Possibilities:
- `_agent-order_` (allowing hyphen in a variable-name would be new)
- `_agent_order_` (ecmarkdown might think the variable-name ends at the middle '_')
- `_agentOrder_` (the usual for multi-word variable-names)
- `_AgentOrder_` (uncommon to start a variable-name with a cap letter, but it's not unheard of, and would match the field name)

With anything other than the first alternative, would we also then change occurrences of "agent-order" *outside* algs? (Alternatively, should the prose refer to the field name rather than the variable-name?)

----
Note that SAB added another addition to the `Let` grammar, but it only appears once, in `Atomics.compareExchange`:
```
1. Let `compareExchange` denote a semantic function ...
```
I wasn't sure what to do about that either.